service/src/main/kotlin/fi/espoo/evaka/messaging/bulletin/BulletinControllerEmployee.kt

```diff
@@ -45,7 +47,8 @@ class BulletinControllerEmployee(
                     content = "",
                     createdByEmployee = user.id,
                     groupId = null,
-                    sentAt = null
+                    sentAt = null,
+                    unitId = unitId
```

Comment:
getDraftBulletins endpoint should take and use `@RequestParam(required = true) unitId: UUID` in same way as getSentBulletins